cmd/operator-sdk/bundle/create.go

```diff
@@ -330,3 +342,68 @@ func copyScorecardConfig() error {
 	}
 	return nil
 }
+
+func addSDKMetricLabels() error {
+	var sdkMetricContent string
+
```

Comment:
Could we remove the spaces between the code lines? 
They are not helpful to understand the code.